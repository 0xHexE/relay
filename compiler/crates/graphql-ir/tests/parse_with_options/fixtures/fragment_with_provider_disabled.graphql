# expected-to-throw
fragment TestFragment on User
  @argumentDefinitions(
    id: {type: "ID"}
    size: {type: "[Int]", defaultValue: [100]}
    # should fail due to provider arguments being a gated feature
    foo: {type: "Boolean!", provider: "test_foo_gk.js"} 
  ) {
  profilePicture(size: $PictureSize) {
    uri
  }
}