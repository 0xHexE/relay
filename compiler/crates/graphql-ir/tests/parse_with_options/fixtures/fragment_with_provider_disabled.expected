==================================== INPUT ====================================
# expected-to-throw
fragment TestFragment on User
  @argumentDefinitions(
    id: {type: "ID"}
    size: {type: "[Int]", defaultValue: [100]}
    # should fail due to provider arguments being a gated feature
    foo: {type: "Boolean!", provider: "test_foo_gk.js"} 
  ) {
  profilePicture(size: $PictureSize) {
    uri
  }
}
==================================== ERROR ====================================
✖︎ Invalid usage of provided variable: this feature is gated and currently set to limited to: arg_with_enabled_provider0, arg_with_enabled_provider1

  fragment_with_provider_disabled.graphql:7:29
    6 │     # should fail due to provider arguments being a gated feature
    7 │     foo: {type: "Boolean!", provider: "test_foo_gk.js"} 
      │                             ^^^^^^^^^^^^^^^^^^^^^^^^^^
    8 │   ) {
